frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2464,7 +2464,14 @@ end
 
 function ReaderFooter:onBookMetadataChanged(prop_updated)
     if prop_updated and (prop_updated.metadata_key_updated == "title" or prop_updated.metadata_key_updated == "authors") then
-        self:updateFooterText()
+        self:maybeUpdateFooter()
```

Comment:
No harm, but no reasons actually I believe.

Reply:
Yeah, that was my read also, just being cautious ;).

Reply:
(It's also a question of setting a precedent: that's the only way things should request a footer refresh, short of very very very specific cases ;)).